packages/cross_file/test/x_file_image_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:typed_data';
+import 'dart:ui';
+
+import 'package:cross_file/cross_file.dart';
+import 'package:cross_file/src/x_file_image.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'image_data.dart';
+
+void main() {
+  test('XFileImage sets tag', () async {
+    final Uint8List bytes = Uint8List.fromList(kBlueSquarePng);
+    final XFile file = XFile('', bytes: bytes);
+    final XFileImage provider = XFileImage(file);
+
+    final MultiFrameImageStreamCompleter completer =
+        provider.load(provider, _decoder) as MultiFrameImageStreamCompleter;
+
+    expect(completer.debugLabel, 'XFileImage(${describeIdentity(file)})');
+  });
+
+  test('Resize image sets tag', () async {
```

Comment:
Similar to https://github.com/flutter/flutter/blob/125545da322048565911bf81118cce60f05036ce/packages/flutter/test/painting/image_provider_test.dart#L179-L188